phalcon/Helper/Str.zep

```diff
@@ -396,6 +396,38 @@ class Str
         return parts[0] . separator. number;
     }
 
+    /**
+     * Removes a number from a string or decrements that number if it already is defined.
+     * defined
+     *
+     * ```php
+     * use Phalcon\Helper\Str;
+     *
+     * echo Str::decrement("a_1");    // "a"
+     * echo Str::decrement("a_2");  // "a_1"
+     * ```
+     *
+     * @param string $text
+     * @param string $separator
+     *
+     * @return string
+     */
+    final public static function decrement(string text, string separator = "_") -> string
```

Comment:
Can you move this method to be after `decapitalize`. Keeping alphabetical sorting for methods.